'makeWebpackPkg' errored... Error in plugin 'webpack-stream', modules not found

## Type of issue
Bug/build failure

## Description
When I go to build a prebid.js file from mac terminal using `gulp build --modules=/path_to_file/pbModules.json` I get a very long error that starts with an error in the *makeWebpackPkg* and *webpack-stream* plugins
```
[10:30:52] Using gulpfile ~/Desktop/Prebid.js/gulpfile.js
[10:30:52] Starting 'build'...
[10:30:52] Starting 'clean'...
[10:30:52] Finished 'clean' after 21 ms
[10:30:52] Starting 'build-bundle-prod'...
[10:30:52] Starting 'makeWebpackPkg'...
[10:31:44] 'makeWebpackPkg' errored after 52 s
[10:31:44] Error in plugin 'webpack-stream'
```

## Steps to reproduce
The last time I used terminal to build prebid was 2018 when my download was version 1.25.
I typed `git pull` to update all the files and received 2.30 as the new version in the files.
Then after typing the command `gulp build --modules=/path_to_file/pbModules.json`, that's when I get these errors.

I also tried redownloading the files before I did `git pull` and this same thing happened.

## Platform details
2015 Macbook Pro, 10.14.5

### Actual results
Here's only the start of the error output since a lot of it repeats and there's quite a lot of it. It mentions a number of modules are not found in directories that don't exist, nor that I see as downloadable on the git homepage for prebid.

```Message:
	./node_modules/destroy/index.js
Module not found: Error: Can't resolve 'fs' in '/Users/admin/Desktop/Prebid.js/node_modules/destroy'
resolve 'fs' in '/Users/admin/Desktop/Prebid.js/node_modules/destroy'
	Parsed request is a module
	using description file: /Users/admin/Desktop/Prebid.js/node_modules/destroy/package.json (relative path: .)
		Field 'browser' doesn't contain a valid alias configuration
	after using description file: /Users/admin/Desktop/Prebid.js/node_modules/destroy/package.json (relative path: .)
		resolve as module
			looking for modules in /Users/admin/Desktop/Prebid.js
				using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: .)
					Field 'browser' doesn't contain a valid alias configuration
				after using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: .)
					using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: ./fs)
						no extension
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/fs doesn't exist
						.js
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/fs.js doesn't exist
						.json
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/fs.json doesn't exist
						as directory
							/Users/admin/Desktop/Prebid.js/fs doesn't exist
			/Users/admin/Desktop/Prebid.js/node_modules/destroy/node_modules doesn't exist or is not a directory
			/Users/admin/Desktop/Prebid.js/node_modules/node_modules doesn't exist or is not a directory
			/Users/admin/Desktop/node_modules doesn't exist or is not a directory
			/Users/admin/node_modules doesn't exist or is not a directory
			/Users/node_modules doesn't exist or is not a directory
			/node_modules doesn't exist or is not a directory
			looking for modules in /Users/admin/Desktop/Prebid.js/node_modules
				using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: ./node_modules)
					Field 'browser' doesn't contain a valid alias configuration
				after using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: ./node_modules)
					using description file: /Users/admin/Desktop/Prebid.js/package.json (relative path: ./node_modules/fs)
						no extension
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/node_modules/fs doesn't exist
						.js
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/node_modules/fs.js doesn't exist
						.json
							Field 'browser' doesn't contain a valid alias configuration
							/Users/admin/Desktop/Prebid.js/node_modules/fs.json doesn't exist
						as directory
							/Users/admin/Desktop/Prebid.js/node_modules/fs doesn't exist```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:7.10
+      - image: circleci/node:8.9.0
       
       # Specify service dependencies here if necessary
       # CircleCI maintains a library of pre-built images

--- a/README.md
+++ b/README.md
@@ -66,7 +66,7 @@ module.exports = {
 }
 ```
 
-Or for Babel 6 and/or Node v8.6.0 and less:
+Or for Babel 6:
 ```javascript
             // you must manually install and specify the presets and plugins yourself
             options: {
@@ -112,7 +112,7 @@ prebid.requestBids({
     $ cd Prebid.js
     $ npm install
 
-*Note:* You need to have `NodeJS` 6.x or greater installed.
+*Note:* You need to have `NodeJS` 8.9.x or greater installed.
 
 *Note:* In the 1.24.0 release of Prebid.js we have transitioned to using gulp 4.0 from using gulp 3.9.1.  To compily with gulp's recommended setup for 4.0, you'll need to have `gulp-cli` installed globally prior to running the general `npm install`.  This shouldn't impact any other projects you may work on that use an earlier version of gulp in it's setup.
 

--- a/gulpHelpers.js
+++ b/gulpHelpers.js
@@ -91,11 +91,11 @@ module.exports = {
     }
     return Object.assign(externalModules.reduce((memo, module) => {
       try {
-        var modulePath = require.resolve(module);
-        // only include modules from our project.  This is in case a dependency shares one of our module's name (like express).
-        if (modulePath.indexOf('/modules/') !== -1) {
-          memo[modulePath] = module;
-        }
+        // prefer internal project modules before looking at project dependencies
+        var modulePath = require.resolve(module, {paths: ['./modules']});
+        if (modulePath === '') modulePath = require.resolve(module);
+
+        memo[modulePath] = module;
       } catch (err) {
         // do something
       }

--- a/gulpHelpers.js
+++ b/gulpHelpers.js
@@ -92,7 +92,10 @@ module.exports = {
     return Object.assign(externalModules.reduce((memo, module) => {
       try {
         var modulePath = require.resolve(module);
-        memo[modulePath] = module;
+        // only include modules from our project.  This is in case a dependency shares one of our module's name (like express).
+        if (modulePath.indexOf('/modules/') !== -1) {
+          memo[modulePath] = module;
+        }
       } catch (err) {
         // do something
       }

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
   "author": "the prebid.js contributors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=4.0"
+    "node": ">=8.9.0"
   },
   "devDependencies": {
     "@babel/core": "^7.2.2",

